projects/synapse/index.js

```diff
@@ -46,13 +31,12 @@ const DATA = {
       metapool: "0x930d001b7efb225613aC7F35911c52Ac9E111Fa9",
       basepool: "0x938aFAFB36E8B1AB3347427eb44537f543475cF9",
     },
-    bridge: "0xd123f70ae324d34a9e76b67a27bf77593ba8749f",
-    bridgeAssets: [
+    tokens: [
+      "0xaa88c603d142c371ea0eac8756123c5805edee03", // DOG
```

Comment:
does any of these tokens have an insane amount bridge that is higher than mcap?

Reply:
No, all tokens bridged by Synapse are backed by actual tokens deposited (or burnt) on the other chain, so it's not possible to bridge more than mcap